mteb/model_meta.py

```diff
@@ -97,7 +98,7 @@ class ModelMeta(BaseModel):
     reference: STR_URL | None = None
     similarity_fn_name: DISTANCE_METRICS | None = None
     use_instructions: bool | None = None
-    zero_shot_benchmarks: list[str] | None = None
+    training_datasets: dict[str, list[str]] | None = None
```

Comment:
Will the dataset name be validated somehow, e.g. must match with one of the existing dataset names within MTEB?

If so, where should that validation be?

Reply:
I think this is hard to validate because there might be datasets intended only for training that overlap with some tasks

Reply:
@Samoed By validation, I'm referring whether the entered dataset name is from the list of tasks in MTEB. Right now, this field accepts any dataset name, e.g. "ABCDE", but such a dataset does not exist in MTEB, and I feel that should an invalid entry in the model meta. Could you elaborate what "that overlap with some tasks" means?

Reply:
For example, if a model is trained on the [NLLB](https://huggingface.co/datasets/allenai/nllb) dataset, some bitext mining datasets might potentially leak into the training data, but it's hard to determine which ones. Similarly, if a model is trained on Wikipedia, many datasets could have overlapping examples in the training data. Identifying these overlaps would require extensive testing to pinpoint where the leaks occur.

Reply:
Would be great to have validation, though I could imagine that people would also add datasets that are not in mteb here